h/api/test/schemas_test.py

```diff
@@ -42,13 +42,13 @@ def test_it_sets_appropriate_error_message_when_data_invalid(self):
 class TestAnnotationSchema(object):
```

Comment:
I'd argue that these are testing that the schema that we've implemented in jsonschema does what we think it does, i.e. testing that we're using jsonschema correctly.